src/main/java/com/rakovets/course/java/core/practice/oop/principles/hero/Archer.java

```diff
@@ -0,0 +1,19 @@
+package com.rakovets.course.java.core.practice.oop.principles.hero;
+
+public class Archer  extends Hero{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"